module/solutions/helper.js

```
@@ -4307,6 +4309,8 @@ function _generateLink(appsPortalBaseUrl, prefix, solutionLink, solutionType) {
     case messageConstants.common.IMPROVEMENT_PROJECT:
       link = appsPortalBaseUrl + prefix + messageConstants.common.CREATE_PROJECT + solutionLink;
       break;
+    case messageConstants.common.COURSE:
+      link = solutionLink
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)